package.json

```
@@ -5,22 +5,24 @@
   "description": "Monorepo root package for lit.dev",
   "author": "Google LLC",
   "license": "BSD-3-Clause",
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
-    "bootstrap": "lerna bootstrap --ci",
     "build": "wireit",
     "start": "run-p -r start:main start:playground start:fake-github",
-    "start:main": "LITDEV_ENV=local MODE=main lerna run start --scope=lit-dev-server --stream",
-    "start:playground": "LITDEV_ENV=local MODE=playground lerna run start --scope=lit-dev-server --stream",
+    "start:main": "LITDEV_ENV=local MODE=main npm run start -w lit-dev-server",
+    "start:playground": "LITDEV_ENV=local MODE=playground npm start -w lit-dev-server",
     "start:fake-github": "wireit",
-    "dev": "cd packages/lit-dev-server && npm run build && cd ../lit-dev-content && (npm run dev:build:assets watch & npm run dev:build:eleventy:watch & node ../lit-dev-tools-esm/lib/dev-server.js --open)",
+    "dev": "cd lit-dev-server && npm run build && cd ../lit-dev-content && (npm run dev:build:assets --watch & npm run dev:build:eleventy:watch & node ../lit-dev-tools-esm/lib/dev-server.js --open)",
```

The `cd` here doesn't work.
```suggestion
    "dev": "cd packages/lit-dev-server && npm run build && cd ../lit-dev-content && (npm run dev:build:assets --watch & npm run dev:build:eleventy:watch & node ../lit-dev-tools-esm/lib/dev-server.js --open)",
```

---

Oops, fixed